content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,70 @@ positional arguments:
 
 ## Description
 
-In order to track parameters and hyperparameters associated to machine learning
-experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. They usually have simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+Parameters are simple values used inside your code to influence the results
```

Comment:
Sometimes values in `params` files are things like dates for an ETL script or paths that get reused throughout the code. Maybe we are focusing too hard on the ML experimentation use case here. Pretty minor, though, so feel free to go with whatever you think works best.

Reply:
Already changes simple -> any in another PR (merged to master and propagated here).